README.md

```diff
@@ -14,13 +14,25 @@ in lightweight virtual machines, called microVMs, which combine the security and
 isolation properties provided by hardware virtualization technology with the
 speed and flexibility of containers.
 
+## Supported platforms
+
+Firecracker officially supports the following CPUs for production usecases:
+- Intel Skylake
+- Intel Cascade Lake
+- AMD Zen2
+
+ARM aarch64 support is currently offered as dev preview.
+
+**Note**: While not officially supported, Firecracker *may* work on other Intel x86_64,
```

Comment:
I don't think we're customer obsessed here.

What we support should be what our users with valid multi-tenant serverless use cases want to use Firecracker on, intersected with what's reasonable given our code base (but we should still be open to broaden that). For example, we don't continuously test on Kaby Lake, but I think it's something we would definitely support if someone would raise an issue.

I would propose a statement like this:

_Firecracker is [generally available](link to support doc) on Intel x86_64 and AMD x86_64 CPUs that offer hardware virtualization support, and that are released starting with 2015. All production use cases should follow [these production host setup instructions]._

_Firecracker is in [developer preview](link to support doc) (and not supported for production workloads) on CPUs based on Arm Cortex-A aarch64 cores that offer hardware virtualization support, and that are released starting with 2015._

_Firecracker may work on other CPUs 64-bit CPUs with support for hardware virtualization, but any such platform is currently not supported and not fit for production. If you want to run Firecracker on such platforms, please open a feature request._